src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java

```diff
@@ -32,6 +32,26 @@
  */
 public class SalesforceConnectorConfig extends PluginConfig {
 
+  @Nullable
```

Comment:
nit: leave a space between each field for better readability.

Reply:
Added